source/javascripts/all.js

```diff
@@ -1,506 +0,0 @@
-			this.el.html("<div class=scroller></div>");
-			$(".scroller", this.el).append(content);
-
-			var index0;
-			var items = this.items();
-			var coords0 = this.coords(items.first());
-
-			items.each(function(index, el){
-				var coords = this.coords($(el));
-				// Tag each Picture
-				var format = this.set_format(el, coords);
-				// Get The minimal Height
-				// to define line_height
-				if (format === this.format[0]) {
-					if (coords0.height > coords.height) {
-						index0 = index;
-					}
-				}
-			}.bind(this));
-
-			// Previous Test didnt Work
-			if (index0 == undefined) {
-				_.find(items, function(el, index) {
-					var test = this.get_format(el) === this.format[0];
-					if (test) index0 = index;
-					return test;
-				}.bind(this));
-			}
-
-			if (index0 == undefined) {
-				index0 = 0;
-				this._clean = false;
-			}
-
-			// Add min-coords
-			var el = $(items.get(index0));
-			var coords = this.coords(el);
-			
-			this._ref = coords;
-			var success = function() {
-				this.resize();
-			}.bind(this);
-
-			$(this.el).trigger("format:end", {success: success});
-		},
-
-		set_nav: function() {
-			if (_.is_touch() || this.el.get(0).scrollWidth === this.el.get(0).offsetWidth) return;
-			
-			// Display Visibility
-			var display_buttons = function() {
-				var min = 0;
-				var max = this.el.get(0).scrollWidth - this.el.get(0).offsetWidth;
-				var value = this.el.get(0).scrollLeft;
-				
-				$("[data-action=next]")[(value === max) ? "addClass" : "removeClass"]("disabled");
-				$("[data-action=back]")[(value === min) ? "addClass" : "removeClass"]("disabled");
-				
-			}.bind(this);
-			
-			var _goto = function(event) {
-				var target = event.currentTarget;
-				var action = $(target).data("action");
-				var step = this.el.width() / 2;
-				var value = (action === "next") ? this.el.get(0).scrollLeft + step : this.el.get(0).scrollLeft - step;
-
-				// ANimation
-				this.el.animate({ "scrollLeft": value}, { complete: display_buttons});
-			}.bind(this);
-
-			// Add navigation
-			this.el.append('<nav><button data-action="back"><button data-action="next"></nav>');
-			_.create_affix($("nav", this.el));
-			
-			// Handle Click Event
-			$("button", this.el).on("click", _goto);
-			
-			// Handle Scroll Event
-			$(this.el).on("scroll", _.throttle(display_buttons, 100))
-			
-			$(this.el).trigger("scroll");
-			$(this.el).off("gallery:resize");
-		},
-
-		replace_picture: function(el, options) {
-			if (el === undefined || !el) {
-				return false;
-			}
-			if (!options) options = {};
-			var style = {
-				"width": options.coords.width,
-				"height": options.coords.height,
-				"background-repeat": "no-repeat",
-				"background-position": "50% 50%",
-				"background-image": "url('" + $(el).attr("src") + "')"
-			};
-
-			// Create a new container
-			// if el is a picture
-			var parent = $(el.parentNode);
-			if (parent.attr("data-format") !== undefined || parent.hasClass("scroller")) {
```

Comment:
FIX : this test was removed into the refactoring (useless for pre-set into HTML attributes)